kafka-connect-s3/src/main/java/io/confluent/connect/s3/auth/AwsAssumeRoleCredentialsProvider.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Confluent Inc., Nordstrom Inc.
```

Comment:
nit: do we mention multiple orgs in the license header?

Reply:
We do not currently. That was missed. Thanks